modules/solid_mechanics/include/materials/SolidModel.h

```diff
@@ -79,8 +79,10 @@ class SolidModel : public Material
   Function * const _poissons_ratio_function;
 
   const CRACKING_RELEASE _cracking_release;
-  const Real _cracking_stress;
+  Real _cracking_stress;
   const Real _cracking_residual_stress;
+  Function * const _cracking_stress_function;
```

Comment:
@permcody - Thanks for your comments. I did not expect the object that the pointer pointing to is constant. I would use it similar to the other material property function pointers such as _youngs_modulus_function and _poissons_ratio_function.